<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateServerDialog</class>
 <widget class="QDialog" name="CreateServerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Game</string>
  </property>
  <layout class="QVBoxLayout" name="dialogLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="GeneralGameSetupPanel" name="generalSetupPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRules">
      <attribute name="title">
       <string>Rules</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="GameRulesPanel" name="rulesPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabServer">
      <attribute name="title">
       <string>Server</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="MiscServerSetupPanel" name="serverPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFlags">
      <attribute name="title">
       <string>Flags</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="DMFlagsPanel" name="dmflagsPanel" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCustomParameters">
      <attribute name="title">
       <string>Custom parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="CustomParamsPanel" name="customParamsPanel" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../../../media/resources.qrc">
         <normaloff>:/icons/media-playback-start.png</normaloff>:/icons/media-playback-start.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCommandLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Obtain command line required to launch this game.</string>
       </property>
       <property name="text">
        <string>Command line</string>
       </property>
       <property name="icon">
        <iconset resource="../../../media/resources.qrc">
         <normaloff>:/icons/terminal-32.png</normaloff>:/icons/terminal-32.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomParamsPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/customparamspanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MiscServerSetupPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/miscserversetuppanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameRulesPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/gamerulespanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DMFlagsPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/dmflagspanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneralGameSetupPanel</class>
   <extends>QWidget</extends>
   <header>gui/createserver/generalgamesetuppanel.h</header>
   <container>1</container>
   <slots>
    <signal>gameModeChanged(GameMode)</signal>
    <signal>pluginChanged(EnginePlugin*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../../media/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>generalSetupPanel</sender>
   <signal>gameModeChanged(GameMode)</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>initGamemodeSpecific(GameMode)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalSetupPanel</sender>
   <signal>pluginChanged(EnginePlugin*)</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>initEngineSpecific(EnginePlugin*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCommandLine</sender>
   <signal>clicked()</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>showStartGameCommandLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStart</sender>
   <signal>clicked()</signal>
   <receiver>CreateServerDialog</receiver>
   <slot>startGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>669</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initGamemodeSpecific(GameMode)</slot>
  <slot>initEngineSpecific(EnginePlugin*)</slot>
  <slot>showStartGameCommandLine()</slot>
  <slot>startGame()</slot>
 </slots>
</ui>
